libraries/stdlib/common/src/kotlin/TextH.kt

```diff
@@ -66,8 +64,75 @@ expect enum class RegexOption {
 // From char.kt
 
 expect fun Char.isWhitespace(): Boolean
-expect fun Char.toLowerCase(): Char
-expect fun Char.toUpperCase(): Char
+
+/**
+ * Converts this character to lower case using Unicode mapping rules of the invariant locale.
+ */
+@OptIn(ExperimentalStdlibApi::class)
+@kotlin.internal.InlineOnly
+public inline fun Char.toLowerCase(): Char = lowercaseChar()
```

Comment:
Better to avoid non-expect functions in this file.
The new functions could also be placed right in `stdlib/src`

Reply:
done